app/dashboard/admin.py

```diff
@@ -143,6 +143,10 @@ def network_link(self, instance):
         return mark_safe(f"<a href={url}>{copy}</a>")
 
 
+class HackathonEventAdmin(admin.ModelAdmin):
+    pass
```

Comment:
Just a random note, you don't have to include this now... but moving forward, we'll want to update this to include a preview of the logo similar to the way we're handling it for avatars.

Reply:
I add this functionality in the next round 👍